### PR TITLE
Adding information on orchestrating multiple services

### PR DESCRIPTION
**Pull request by bnookala**
Just a start.

Related to #359

### CODE REVIEW

File: gitops/azure-devops/MultipleServices.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+> *Please first overview the [Pipeline Thinking](./PipelineThinking.md) document before continuing through this document.*
+
+## Overview: GitOps
+
+In our prescribed gitops workflow, you are expected to push changes to your cluster through changes to a HLD repository.
+
+Recall the prescribed workflow that encompasses the manifest yaml generation pipeline:
+
+![Example of manifest yaml generation pipeline](images/manifest-gen.png)
```
Comment (samiyaakhtar) [inline gitops/azure-devops/MultipleServices.md]: Unable to load this image?

**Comment by andrebriggs** [comment]
Will take a look later today

File: gitops/azure-devops/MultipleServices.md (hunk 1 above)
Comment (bnookala) [reply gitops/azure-devops/MultipleServices.md]: might be the absolute path vs a relative path

**Comment by bnookala** [comment]
@samiyaakhtar @andrebriggs I think the images should load now

**Pull request merged**